requirements.txt

```diff
@@ -2,3 +2,4 @@ Click
 requests
 semver
 wheel
+tomli; python_version < "3.11"

```

Comment:
`<` or `<=` 3.11?

Reply:
< is correct, tomllib was added in 3.11.